azure-pipelines.yml

```diff
@@ -558,3 +558,56 @@ stages:
           inputs:
             testResultsFiles: '**/test-*.xml'
             testRunTitle: 'Test results for macOS Python $(python.version)'
+    - job: 'Tutorials'
+      pool: {vmImage: 'ubuntu-latest'}
+      strategy:
+        matrix:
+          Python38:
+            python.version: '3.8'
+      variables:
+        QISKIT_SUPPRESS_PACKAGING_WARNINGS: Y
+        PIP_CACHE_DIR: $(Pipeline.Workspace)/.pip
+      steps:
+        - task: UsePythonVersion@0
+          inputs:
+            versionSpec: '$(python.version)'
+          displayName: 'Use Python $(python.version)'
+        - task: Cache@2
+          inputs:
+            key: 'pip | "$(Agent.OS)" | "$(python.version)"'
+            restoreKeys: |
+              pip | "$(Agent.OS)"
+              pip
+            path: $(PIP_CACHE_DIR)
+          displayName: Cache pip
+        - bash: |
+            set -e
+            git clone https://github.com/Qiskit/qiskit-tutorials --depth=1
+            python -m pip install --upgrade pip
+            pip install -U -r requirements.txt -r requirements-dev.txt -c constraints.txt
+            pip install -c constraints.txt -e .
+            pip install "qiskit-ibmq-provider" "qiskit-aer" "z3-solver" "qiskit-ignis" "qiskit-aqua" "pyscf<1.7.4" "matplotlib<3.3.0" sphinx nbsphinx sphinx_rtd_theme cvxpy -c constraints.txt
+            python setup.py build_ext --inplace
+            sudo apt install -y graphviz pandoc
+            pip check
+          displayName: 'Install dependencies'
+        - bash: |
+            set -e
+            cd qiskit-tutorials
+            rm -rf tutorials/chemistry/*ipynb
+            rm -rf tutorials/finance/*
+            rm -rf tutorials/optimizations/*
```

Comment:
Is this for runtime? The job on qiskit-tutorials looks like it takes ~15m.

Reply:
Yeah I did it for run time, but I think that you're right and it might not be necessary. Earlier versions of this pr were hitting timeouts, but that was unrelated and because of a pulse drawer issue with matplotlib 3.3.0

Reply:
I added them back in: https://github.com/Qiskit/qiskit-terra/pull/4907/commits/25290ebb4b5a7defab132a60dedd7a3a14534f5c les see if it takes too long, and if it does we can always revert that before merging.

Reply:
It looks like the job with all the notebooks took 22min: https://dev.azure.com/qiskit-ci/qiskit-terra/_build/results?buildId=17764&view=logs&jobId=6d8ff735-f981-52fa-2234-caef37f9a56b ~18min for the actual notebook execution/sphinx build.